packages/modules/fronius/device.py

```diff
@@ -89,7 +92,8 @@ def update(self) -> None:
                         break
```

Comment:
Wird die Korrektur des EVU-Zähler noch benötigt, wenn die Einbauposition ermittelt wird und dann der entsprechende Wert ausgelesen wird?
In der update-Methode ist die Verechnung aus dem alten Modul enthalten, aber in der read_legacy-Methode habe ich es nicht beachtet. Dort werden EVU, WR und Speicher getrennt ausgelesen und bei der obigen Verrechnung ist die WR-Leistung immer 0, wenn der EVU-Zähler ausgelesen wird.
Im Forum hat keiner falsche Werte gemeldet und auch der Testdatensatz [https://github.com/vuffiraa72/openWB/blob/8ee4b974a64257554a2d76e667fe95cbe362da14/packages/modules/fronius/counter_sm_test.py#L341](url) legt nahe, dass Fronius das berücksichtigt.

Reply:
Interessante Frage, diese Logik wurde lange nicht hinterfragt.
Bei meiner Installation, Smartmeter im Verbrauchszweig, führt die Korrektur zu einem Vorzeichenwechsel. Die absoluten Werte bleiben gleich.
Hier kann aufgeräumt werden, es hat aber nicht so viel mit dem Inhalt dieses PR zu tun. Ich wurde das gerne getrennt angehen, da man sich auch ein paar Gedanken über Testbarkeit und Auswirkungen auf die 2.0 machen muss.

Reply:
Ok, dann nehme ich den PR erstmal so rein.